build_project/local.go

```diff
@@ -0,0 +1,55 @@
+package build
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/tarantool/cartridge-cli/common"
+	"github.com/tarantool/cartridge-cli/project"
+)
+
+func buildProjectLocally(projectCtx *project.ProjectCtx) error {
+	checkLocalBuildRecommendedBinaries()
+
+	// pre-build
+	preBuildHookPath := filepath.Join(projectCtx.BuildDir, preBuildHookName)
+
+	if _, err := os.Stat(preBuildHookPath); !os.IsNotExist(err) {
+		log.Infof("Running `%s`", preBuildHookName)
+		err = common.RunHook(preBuildHookPath, !projectCtx.Quiet)
+		if err != nil {
+			return fmt.Errorf("Failed to run pre-build hook: %s", err)
+		}
+	}
+
+	// tarantoolctl rocks make
+	log.Infof("Running `tarantoolctl rocks make`")
+	rocksMakeCmd := exec.Command("tarantoolctl", "rocks", "make")
+	err := common.RunCommand(rocksMakeCmd, projectCtx.BuildDir, !projectCtx.Quiet)
+	if err != nil {
+		return fmt.Errorf("Failed to install rocks: %s", err)
+	}
+
+	return nil
+}
+
+func checkLocalBuildRecommendedBinaries() {
+	var recommendedBinaries = []string{
```

Comment:
So, I think we should check also "required" binaries here.

Reply:
Yes, I'm thinking about doing that in `project.FillCtx` function because it's called for every command.

Reply:
Do it somewhere. But we should add some validation before run every command. It makes us a bit user-friendly

Reply:
Added `project.CheckTarantoolBinaries()`, on build it causes an erro, on create just warning.